lib/martian_robot.rb

```diff
@@ -1,25 +1,30 @@
+require_relative 'movements'
+require_relative 'mars'
+
 class MartianRobot
 
   attr_reader :coordinates, :orientation
 
-  def initialize(coordinates, orientation)
+  def initialize(coordinates, orientation, size)
```

Comment:
The size is a variable specific to the planet and not to the `MatianRobot`.

Maybe the robot should receive a planet as a variable instead of a size.